docs/guides/deploy.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Deploying your first project with Nitric Deploy
+description: Nitric Deploy builds and deploys the right infrastructure from your code in your chosen cloud.
+---
+
+## What we'll be doing
+
+1. Sign to Nitric Deploy
+2. Create an organization
+3. Set up AWS credentials
+4. Create or import a project
+5. Create an environment
+6. Configure environment variables
+7. Deploy your project
+8. Check out the logs
+
+<br/>
+
+## Sign to Nitric Deploy
+
+> Nitric Deploy is currently in Alpha, you'll need to [request access](https://deploy.nitric.io/signup).
+
+> Your email address will need to match your github account.
+
+Nitric Deploy can be accessed by navigating to https://deploy.nitric.io - you'll be greeted by our sign in screens. Follow the prompts to sign in with GitHub.
+
+<img alt="sign in splash" src="../../assets/img/guides/deploy/sign-in.jpg" height="337.5" />
+<img alt="sign in github" src="../../assets/img/guides/deploy/sign-in-3.jpg" height="337.5" />
+
+<br/>
+
+## Create an organization
+
+Organizations allow you to group your projects. For example, I have an organization which contains all of my demo projects.
+
+Lets create a new organization and give it a name.
+
+<img alt="organization splash" src="../../assets/img/guides/deploy/add-org.jpg" height="337.5" />
+<img alt="create organization" src="../../assets/img/guides/deploy/add-org-2.jpg" height="337.5" />
+
+<br/>
+
+## Set up AWS credentials
+
+The next step is to set up our AWS credentials with our new organization. Head to settings -> Credentials and click 'Create Credential'.
+
+You'll need to name your credentials and then set your AWS credentials. 
+
+> If you're unsure about what these are or how to get them [click here](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html).
+
+> Only valid credentials will work here.
+
+<img alt="credentials splash" src="../../assets/img/guides/deploy/add-org-credentials.jpg" height="337.5" />
+<img alt="create credentials" src="../../assets/img/guides/deploy/add-org-credentials-2.jpg" height="337.5" />
+
+<br/>
+
+## Create or import a project
+
+Projects can be imported from existing Git repositories or we can scaffold a new project for you from one our of templates.
+
+<img alt="project splash" src="../../assets/img/guides/deploy/add-project.jpg" height="337.5" />
+<img alt="import project" src="../../assets/img/guides/deploy/add-project-2.jpg" height="337.5" />
+
+### Existing projects
+
+Existing projects have either been scaffolded by Nitric Deploy or have been created using the Nitric CLI.
```

Comment:
This might be just non-technical person feedback, but this sentence isn't totally clear to me.

Reply:
Is it "In order to import an existing project, it must have been either scaffolded by..." ?